### PR TITLE
Privacy notice for About

### PR DESCRIPTION
**Pull request by ElenaFdR**
This fulfills part of #318.

### CODE REVIEW

File: organization/privacy.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+We won't share it or sell it.
+
+## Data retention
+
+People can always unsubscribe from our mailing list.
+
+As a contact for a piece of work, we may contact you where a work matter requires your input. Once you leave the role, we will update or delete the information we hold about you. This will take place at least once a year.
+
+We keep staff data for up to seven years, as required by Dutch law.
```
Comment (ElenaFdR) [inline organization/privacy.md]: @bvhme thinks this might be 10 years for calendars. Check!

**Comment by ElenaFdR** [comment]
Travis should fail because I have a link to the future home of this page on Github.

**Comment by bvhme** [comment]
And, I think this is ready to merge, and Travis is failing for all the right reasons.

@ElenaFdR shall I merge this (by pressing my big red merge button)?

**Comment by ElenaFdR** [comment]
Yup, that sounds great!

**Pull request merged**